static/src/js/actions/project.js

```diff
@@ -182,13 +182,13 @@ export const getProjectCollections = () => async (dispatch, getState) => {
   }
 
   // Fetch the saved access configurations for the project collections
-  let savedAccessConfigs
+  let savedAccessConfigs = {}
   try {
     const savedAccessConfigsRequestObject = new SavedAccessConfigsRequest(
       authToken,
       earthdataEnvironment
     )
-    savedAccessConfigsRequestObject.search(
+    await savedAccessConfigsRequestObject.search(
```

Comment:
I think if we are using async/await syntax here, we probably can convert this whole block over to it (instead of leaving Promise/then), such as
```js
const response = await savedAccessConfigsRequestObject.search({ collectionIds: filteredIds })  

savedAccessConfigs = response.data
```

but this is just a minor suggestion and doesn't need to block approval